Dockerfile

```diff
@@ -21,17 +21,17 @@ ENV APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=DontWarn
 RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 9386B48A1A693C5C && \
     wget -q -O - https://repo.radeon.com/rocm/rocm.gpg.key | apt-key add -
 
-RUN wget https://repo.radeon.com/amdgpu-install/5.3/ubuntu/focal/amdgpu-install_5.3.50300-1_all.deb  --no-check-certificate
+RUN wget https://repo.radeon.com/amdgpu-install/5.4.3/ubuntu/focal/amdgpu-install_5.4.50403-1_all.deb  --no-check-certificate
 RUN apt-get update && \
 DEBIAN_FRONTEND=noninteractive apt-get install -y --allow-unauthenticated \
-    ./amdgpu-install_5.3.50300-1_all.deb
+    ./amdgpu-install_5.4.50403-1_all.deb
```

Comment:
[Warning] Dockerfile changes are not tested in our CI. Merging them may lead to issues for the clients who use Dockerfile for building/testing environment.

Reply:
I have rebuilt these CI docers and pushed to docker hub, now testing if they work fine and will eventually replace CI docker with these new ones. 
rocm/miopen:ci_rocm543_ubuntu2004 
https://hub.docker.com/layers/rocm/miopen/ci_rocm543_ubuntu2004/images/sha256-6cf15b96ae9363a5e3517112e634a14a53458057a0866266c8c93833287de195?context=explore

Reply:
@junliume Thanks. So it seems like you are following the proposal described at https://github.com/ROCmSoftwarePlatform/MIOpen/pull/1897#discussion_r1044798898, right?

Reply:
@junliume I highly recommend the following steps:
1. Replace docker image on CI
2. Run develop once with it and make sure it passes
3. Merge this PR (i.e. hold off merging until step 2 is complete).